tests/test_plugins/test_resonance_finder.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+import numpy as np
+import pydantic
+
+import gdspy
+import tidy3d as td
+
+from numpy.random import default_rng
+
+from tidy3d.plugins import ResonanceFinder
+
```

Comment:
I ran test coverage using  (note might need to pip install `pytest-cov`)
```
pytest tests --cov-report=html --cov=tidy3d
open htmlcov/index.html
```
This has good coverage of the main class, but the `run_field_time` is not run. Could we add a simple test for it? thanks